src/tuple.h

```diff
@@ -49,7 +49,7 @@
 
 #include "protocol.h"
 
-enum { TUPLE__ROW = 1, TUPLE__PARAMS };
+enum { TUPLE__ROW = 1, TUPLE__PARAMS, TUPLE__PARAMS32 };
```

Comment:
For future readers, please can you include comments explaining what these formats are? In particular the difference between `TUPLE__PARAMS` and `TUPLE__PARAMS32` should be explained here.